package.json

```diff
@@ -40,7 +40,7 @@
     "semver": "5.4.1",
     "triton-metrics": "1.0.3",
     "triton-netconfig": "1.5.0",
-    "ufds": "1.7.0",
+    "ufds": "git+https://github.com/TritonDataCenter/node-ufds.git#8b64c76422819a34e99cda5421e8931892640881",
```

Comment:
So node-ufds needs to land first.  It probably is a NOP w/o all of these other ones, right?

Reply:
Yep, this also depends on TritonDataCenter/node-ufds/pull/12